recipes/libpointmatcher/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class LibpointmatcherConan(ConanFile):
+    name = "libpointmatcher"
+    description = "An Iterative Closest Point (ICP) library for 2D and 3D mapping in Robotics"
+    license = "BSD-3-Clause"
+    topics = ("robotics", "lidar", "point-cloud")
+    homepage = "https://github.com/ethz-asl/libpointmatcher"
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openmp": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_openmp": False,
+    }
+
+    @property
+    def _min_cppstd(self):
+        return "11"
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def requirements(self):
+        self.requires("boost/1.81.0", transitive_headers=True)
+        # FIXME: eigen 3.4.0 is not compatible yet (see https://github.com/ethz-asl/libpointmatcher/issues/496)
+        self.requires("eigen/3.3.9", transitive_headers=True)
```

Comment:
Why is `force=True` not a solution for the version conflicts here until the issue is addressed upstream?

Reply:
Ah of course, I see now

Reply:
I think it's worth discussing this case with the team because I'm afraid of having many more cases like this one.